Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -276,6 +276,9 @@ async function main(): Promise<void> {
 
                 if (artifact.resource.type.toLowerCase() === "container") {
                     var handler = new webHandlers.PersonalAccessTokenCredentialHandler(accessToken);
+                    // this variable uses to force enable zip download option, it is used only in test purpose and shouldn't be used for other reasons
```

Comment:
Maybe it will be useful for customers, but as I see there no customers' requests for this feature. So my opinion is that we should mark currently it as an option for internal use only and do not suggest the use of this option by customers.
If we will have customers' requests for this feature, I assume it will be better to implement it as an optional input, instead of an env variable..